templates/postgresql_database.yml.erb

```diff
@@ -4,9 +4,14 @@ development: &default
   encoding: utf8
   host: localhost
   min_messages: warning
-  pool: 2
+  pool: <%%= ENV.fetch("DB_POOL", 2) %>
+  reaping_frequency: <%%= ENV.fetch("DB_REAPING_FREQUENCY", 10) %>
   timeout: 5000
 
 test:
   <<: *default
   database: <%= app_name %>_test
+
+production:
+  url:  <%%= ENV.fetch("DATABASE_URL") %>
+  pool: <%%= ENV.fetch("DB_POOL", ENV.fetch("MAX_THREADS", 5) %>
```

Comment:
I saw that and didn't change anything. I upped this from what we previously had, 2, and added the second ENV variable.

I also don't think we want production to use the defaults, in case they override something Heroku sets in production.

If anything, possibly this:

```
production: &deploy
  <<: *default # has the pool
  url: <%%= ENV.fetch("DATABASE_URL") %>

staging:
  <<: *deploy
```

@schneems ?